videos_controller_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
```

Comment:
What is this file about? Why was it committed? Why do you call it `..._test.rb` ?